setup.py

```diff
@@ -33,5 +33,6 @@
     extras_require={
         # We can add the mesos specific dependencies here
         'mesos_executor': ['pymesos'],
+        'metrics': ['yelp-meteorite', 'mock']
```

Comment:
We are returning a mock object in metrics if yelp_meteorite isn't around, so that functions called on counters/timers won't break. Thought about trying to create a class with the combined protocol of y_m's counters/timers/gauges, but it might fall short. Thoughts?